tests/foreman/ui/test_errata.py

```diff
@@ -880,7 +880,9 @@ def test_positive_apply_for_all_hosts(
         workflow='deploy-template',
         host_class=ContentHost,
         _count=num_hosts,
-        deploy_network_type='ipv6' if settings.server.is_ipv6 else 'ipv4',
+        # TODO(shwsingh): this is best effor for dualstack. This host deployment
+        # should be a part of a fixture
+        deploy_network_type=settings.content_host.attibutes.network_type,
```

Comment:
There is a typo (against MR 1673)
```suggestion
        deploy_network_type=settings.content_host.attributes.network_type,
```

Reply:
Resolved in f6d046cea517254d299b725ce18486a260a66649. Thanks for spotting it.